src/components/screens/delegates/votingHeader.css

```diff
@@ -2,14 +2,154 @@
 @import './../../../app/mixins.css';
 
 .wrapper {
-  display: flex;
-  justify-content: space-between;
   width: 100%;
-  margin-bottom: 32px;
-  margin-top: 15px;
+  height: 112px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  box-sizing: border-box;
+  position: relative;
+  z-index: 2;
+
+  & .stickyContent {
+    position: relative;
+    height: 78px;
+    z-index: 9;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  & .bg {
+    position: fixed;
+    width: 100%;
+    height: 78px;
+    left: 0;
+    top: 73px;
+    z-index: 0;
+    border-bottom: solid 1px #e1e3eb;
+    background-color: var(--color-white);
+    opacity: 0;
+    transition: all ease 250ms;
+  }
 
-  & > * {
+  &.sticky {
+    & .stickyContent {
+      position: fixed;
+      top: 73px;
+      width: 100%;
+      padding: 0 26px 0 24px;
+      margin-left: -24px;
+      box-sizing: border-box;
+      max-width: 1368px;
+    }
+
+    & .bg {
+      opacity: 1;
+      z-index: 1;
+    }
+  }
+}
+
+.info {
+  display: flex;
+  flex-flow: row nowrap;
+
+  & > div {
     display: flex;
+    flex-flow: column wrap;
+    height: 40px;
+    width: 130px;
+    padding: 19px 20px;
+
+    & .icon {
+      width: 22px;
+      height: 100%;
+      flex: 1 100%;
+      align-items: center;
+      display: flex;
+      margin: 0;
+
+      & img {
+        width: 100%;
+      }
+    }
+
+    & > span,
+    & h5 {
```

Comment:
Ali, I think you don't know this, but according to the css rules that we have in the docs, it is not a big deal, but just for you know about it.

So in this particular case div and span need a class but the h5 for example is ok.

> We should use cascading css for semantic elements (h1, h2, ul, li,...) but not for generic elements (div and span) - if you need to modify their styles, add a new className to them.

Reply:
Done